src/phoenix/experimental/evals/templates/template.py

```diff
@@ -16,81 +14,101 @@
 
 
 @dataclass
+class PromptOptions:
+    provide_explanation: bool = False
+
+
 class PromptTemplate:
-    text: str
+    base_template: str
     variables: List[str]
 
-    def __init__(self, text: str, delimiters: List[str] = [DEFAULT_START_DELIM, DEFAULT_END_DELIM]):
-        self.text = text
-        self._start_delim, self._end_delim = self._get_delimiters(delimiters)
-        self._validate()
-        self.variables = self._parse_variables()
-
-    def format(self, variable_values: Dict[str, Union[bool, int, float, str]]) -> str:
-        prompt = self.text
+    def __init__(
+        self,
+        base_template: str,
```

Comment:
let's at least make the constructor signature be self-explanetory - we can map it to base_template for inheritance purposes.